packages/now-build-utils/test/unit.builds-and-routes-detector.test.ts

```diff
@@ -1891,6 +1895,65 @@ describe('Test `detectApiDirectory`', () => {
   });
 });
 
+describe('Test `detectApiExtensions`', () => {
+  it('should be `null` with no config', async () => {
+    const builders = [
+      {
+        use: '@now/node',
+        src: 'api/**/*.js',
+        config: {
+          zeroConfig: true,
+        },
+      },
+      {
+        use: '@now/python',
+        src: 'api/**/*.py',
+        config: {
+          zeroConfig: true,
+        },
+      },
+      {
+        use: '@now/go',
+        src: 'api/**/*.go',
+        config: {
+          zeroConfig: true,
+        },
+      },
+      {
+        use: '@now/ruby',
+        src: 'api/**/*.rb',
+        config: {
+          zeroConfig: true,
+        },
+      },
+      {
+        use: 'now-bash',
+        src: 'api/index.sh',
+        // No zero config so it should not be added
+      },
+      {
+        use: 'now-rust@1.0.1',
+        src: 'api/user.rs',
+        config: {
+          zeroConfig: true,
+          functions: {
+            'api/**/*.rs': {
+              runtime: 'now-rust@1.0.1',
+            },
+          },
+        },
```

Comment:
@AndyBitz  I deployed a rust project and found that the `src` is actually not a pattern but every file gets its own build. So I made sure this test uses the same input.